- OpenBSD porting glue
- allow the gmake test to find gmake
Index: configure.ac
--- configure.ac.orig
+++ configure.ac
@@ -12,8 +12,13 @@ AC_REVISION([$Revision: 1.6 $])
 AC_SUBST([PACKAGE_VERSION])
 AC_DEFINE_UNQUOTED([WCSLIB_VERSION], [$PACKAGE_VERSION], [Define wcslib version])
 
-# Library version number, same as package version.
-LIBVER="$PACKAGE_VERSION"
+# Library version number. Usually same as package version,
+# but could be different on some OSes, e.g., OpenBSD.
+if test "x${LIBwcs_VERSION}" = x ; then
+  LIBVER="$PACKAGE_VERSION"
+else
+  LIBVER="$LIBwcs_VERSION"
+fi
 AC_SUBST([LIBVER])
 
 AC_CONFIG_SRCDIR([C/wcs.h])
@@ -319,6 +324,12 @@ if test "x$ac_cv_c_compiler_gnu" = xyes ; then
       SHRLD="$SHRLD -shared -Wl,-h\$(SONAME)"
       SHRLN="libwcs.dll"
       ;;
+    openbsd*)
+      SHRLIB="libwcs.so.$LIBVER"
+      SONAME="$SHRLIB"
+      SHRLD="$SHRLD -shared -Wl,-h\$(SONAME)"
+      SHRLN=
+      ;;
     *)
       # Covers Linux and Solaris at least.
       SHRLIB="libwcs.so.$LIBVER"
@@ -343,7 +354,7 @@ AC_PROG_INSTALL
 
 # Older versions of GNU make do not have the -O option, which only facilitates
 # legibility of the output from parallel builds (make -j).
-make --help | grep '\-O' >/dev/null 2>&1 && MAKEFLAGS="-Otarget"
+gmake --help | grep '\-O' >/dev/null 2>&1 && MAKEFLAGS="-Otarget"
 AC_SUBST([MAKEFLAGS])
 
 AC_MSG_NOTICE([End of primary configuration.
